src/v2i-hub/CDASimAdapter/CMakeLists.txt

```diff
@@ -17,4 +17,4 @@ SET(SOURCES ${TEST_SOURCES} WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}/test)
 ADD_EXECUTABLE(${BINARY} ${TEST_SOURCES})
 ADD_TEST(NAME ${BINARY} COMMAND ${BINARY})
 TARGET_INCLUDE_DIRECTORIES(${BINARY} PUBLIC /usr/local/lib src/) 
-TARGET_LINK_LIBRARIES(${BINARY} PUBLIC ${PROJECT_NAME}_lib gtest gmock tmxutils ::carma-clock jsoncpp) 
\ No newline at end of file
+TARGET_LINK_LIBRARIES(${BINARY} PUBLIC ${PROJECT_NAME}_lib gtest gmock gtest_main) 
```

Comment:
Are carma-clock and jsoncpp no longer needed?

Reply:
They are inherited from the library public dependencies